x-pack/plugins/security_solution/public/overview/components/detection_response/cases_by_status/use_cases_by_status.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+import { useState, useEffect } from 'react';
+import { APP_ID } from '../../../../../common/constants';
+import { useGlobalTime } from '../../../../common/containers/use_global_time';
+import { useKibana } from '../../../../common/lib/kibana';
+
+export interface CasesCounts {
+  count_open_cases?: number;
+  count_in_progress_cases?: number;
+  count_closed_cases?: number;
+}
+
+export interface UseCasesByStatusProps {
+  skip?: boolean;
+}
+
+export interface UseCasesByStatusResults {
+  closed: number;
+  inProgress: number;
+  isLoading: boolean;
+  open: number;
+  totalCounts: number;
+  updatedAt: number;
+}
+
+export const useCasesByStatus = ({ skip = false }) => {
+  const {
+    services: { cases },
+  } = useKibana();
+  const { to, from } = useGlobalTime();
+
+  const [updatedAt, setUpdatedAt] = useState(Date.now());
+  const [isLoading, setIsLoading] = useState(true);
+  const [casesCounts, setCasesCounts] = useState<CasesCounts | null>(null);
+
+  // This is a known issue of cases api, it doesn't accept date time format atm
+  // Once they fix this problem we can remove this two lines
+  const fromDate = moment(from).format('YYYY-MM-DD');
```

Comment:
This is a known issue of cases api, it doesn't accept date time format e.g.: 2011-10-05T14:48:00.000Z
It takes date only format 2011-10-05

The api fails on this scenario:
http://localhost:5601/api/cases/_find?status=open&from=2022-04-19T16:00:00.000Z&to=2022-04-20T15:59:59.999Z&owner=securitySolution